samples/net/lwm2m_client/README.rst

```diff
@@ -69,59 +69,51 @@ Build the lwm2m-client sample application for QEMU like this:
    :compact:
 
 The sample will start and automatically connect to the Leshan Demo Server with
-both an IPv4 client endpoint (qemu_x86-ipv4-########) and an IPV6 client
-endpoint (qemu_x86-ipv6-########).  The "########" portion is a randomly
-generated number so you can have multiple clients running at the same time.
+an IPv6 client endpoint "qemu_x86".
+
+To change the sample to use IPv4 instead, set the following in prj.conf:
```

Comment:
To have this formatted better, use this:

    To change the sample to use IPv4, disable IPv6 by changing these two configurations in ``prj.conf``::

       CONFIG_NET_IPV6=n
       CONFIG_NET_APP_NEED_IPV6=n

Reply:
Thanks!  My reStructuredText-fu is quite bad.